datayoga/block.py

```diff
@@ -46,7 +46,7 @@ def init(self):
         """
         pass
 
-    def transform(self, data: Any, context: Context = None) -> Any:
+    def transform(self, data: List[Dict[str, Any]], context: Context = None) -> Any:
```

Comment:
shouldn't it also return List[Dict[str, Any]]?

Reply:
yes. changed.